crates/egui_plot/src/items/mod.rs

```diff
@@ -792,7 +836,25 @@ pub struct Points {
     pub(super) stems: Option<f32>,
 }
 
-impl Points {
+impl<T> Points<T>
+where
+    T: GenericPlotPoints,
+{
+    pub fn new_generic(t: T) -> Self {
```

Comment:
This should be called something like `from_iter`, and have a docstring

Reply:
I can implement FromIterator for Points<T>?

> method `from_iter` can be confused for the standard trait method `std::iter::FromIterator::from_iter`
consider implementing the trait `std::iter::FromIterator` or choosing a less ambiguous method name
for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#should_implement_trait
`-W clippy::should-implement-trait` implied by `-W clippy::all`